SPECS/net-snmp/net-snmp.spec

```diff
@@ -122,6 +123,9 @@ popd
 %{_localstatedir}/run/net-snmp
 
 %changelog
+* Tue Apr 23 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 5.9.4-1
+- Auto-upgrade to 5.9.4 - Fixes for CVE-2022-44792 and CVE-2022-44793
```

Comment:
Change looks OK but these CVEs are medium severity - why do we need to publish them through fast-track?